doc/docs/Python_User_Interface.md

```diff
@@ -1158,7337 +1335,6462 @@ If `match_frequency=False` or `kpoint` is not zero in the given `direction`, the
-                               eig_tolerance=1e-12,
-                               kpoint_func=None,
-                               direction=-1):
+def get_eigenmode_freqs(f):
 ```
 
-<div class="method_docstring" markdown="1">
-
-Given a flux object and list of band indices, return a `namedtuple` with the
-following fields:
-
-+ `alpha`: the complex eigenmode coefficients as a 3d NumPy array of size
-  (`len(bands)`, `flux.nfreqs`, `2`). The last/third dimension refers to modes
-  propagating in the forward (+) or backward (-) directions.
-+ `vgrp`: the group velocity as a NumPy array.
-+ `kpoints`: a list of `mp.Vector3`s of the `kpoint` used in the mode calculation.
-+ `kdom`: a list of `mp.Vector3`s of the mode's dominant wavevector.
-+ `cscale`: a NumPy array of each mode's scaling coefficient. Useful for adjoint
-  calculations.
+<div class="function_docstring" markdown="1">
 
-See [Mode Decomposition](#mode-decomposition)
+Given a flux object, returns a list of the frequencies that it is computing the
+spectrum for.
 
 </div>
 
-<a id="Simulation.get_epsilon"></a>
 
-**Simulation.get\_epsilon**
 
-```python
-def get_epsilon(self, frequency=0, omega=0):
-```
+### Energy Density Spectra
 
-<div class="method_docstring" markdown="1">
+Very similar to flux spectra, you can also compute **energy density spectra**: the energy density of the electromagnetic fields as a function of frequency, computed by Fourier transforming the fields and integrating the energy density:
 
+$$ \frac{1}{2}ε|\mathbf{E}|^2 + \frac{1}{2}μ|\mathbf{H}|^2 $$
 
+The usage is similar to the flux spectra: you define a set of [`EnergyRegion`](#EnergyRegion) objects telling Meep where it should compute the Fourier-transformed fields and energy densities, and call `add_energy` to add these regions to the current simulation over a specified frequency bandwidth, and then use `display_electric_energy`, `display_magnetic_energy`, or `display_total_energy` to display the energy density spectra at the end. There are also `save_energy`, `load_energy`, and `load_minus_energy` functions that you can use to subtract the fields from two simulation, e.g. in order to compute just the energy from scattered fields, similar to the flux spectra. The function used to add an [`EnergyRegion`](#EnergyRegion) is as follows:
 
-</div>
 
-<a id="Simulation.get_epsilon_point"></a>
+<a id="Simulation.add_energy"></a>
 
-**Simulation.get\_epsilon\_point**
+<div class="class_members" markdown="1">
 
 ```python
-def get_epsilon_point(self, pt, frequency=0, omega=0):
+def add_energy(self, *args):
 ```
 
 <div class="method_docstring" markdown="1">
 
-Given a frequency `frequency` and a `Vector3` `pt`, returns the average eigenvalue
-of the permittivity tensor at that location and frequency. If `frequency` is
-non-zero, the result is complex valued; otherwise it is the real,
-frequency-independent part of ε (the $\omega    o\infty$ limit).
+add_energy(fcen, df, nfreq, freq, EnergyRegions...)
```

Comment:
Perhaps `add_energy(fcen, df, nfreq, freq, EnergyRegions...)` should appear in code blocks rather an prose?

Reply:
The best fix for things like this would be to change the code so it specifies the parameters instead of using`*args,**kwargs` but that's probably a task for another day. In any case, that text is part of the docstring so it can just be edited there.

I tried with a triple-backticks to get a full code block but that doesn't look very good next to the one already there, so I'll just wrap them with the single backticks for now.

Another possibility would be to create some way that the tool can recognise a different args string specified within the docstring, and put that in the document instead of, or maybe in addition to, the `def ...` block that is generated automatically. That would probably be best done in a separate PR though.